Gemfile.lock

```diff
@@ -365,7 +365,7 @@ DEPENDENCIES
   database_cleaner (= 1.5.1)
   factory_girl
   foreman (= 0.74.0)
-  gds-api-adapters (= 32.1.0)
+  gds-api-adapters (= 36.0.1)
```

Comment:
yup, that's true. I think otherwise that's the only thing that needs work to make it work